sdk/servicebus/azure-servicebus/tests/async_tests/mgmt_tests/test_mgmt_queues_async.py

```diff
@@ -310,6 +312,8 @@ async def test_async_mgmt_queue_update_success(self, servicebus_namespace_connec
             assert queue_description.lock_duration == datetime.timedelta(seconds=13)
             assert queue_description.max_delivery_count == 14
             assert queue_description.max_size_in_megabytes == 3072
+            assert queue_description.forward_to.endswith(".servicebus.windows.net/{}".format(queue_name))
```

Comment:
is there a reason why we're using `endswith` for comparison not exact matching

Reply:
Yep!  non-live-test replacement.  Otherwise, when in recorded mode, you get a different namespace name (since the replacer doesn't swap the properties, and I didn't feel like writing a new one for this case as validating the topic should be sufficient to assert success.)

Reply:
oob discussion: adding a comment in the code explaining the reason